examples/mip/multiscenario.py

```diff
@@ -0,0 +1,42 @@
+from symfit import MIP, IndexedBase, Eq, Idx, Parameter, symbols, Sum
+import numpy as np
+
+# Warehouse demand in thousands of units
+data_demand = np.array([15, 18, 14, 20])
+
+# Plant capacity in thousands of units
+data_capacity = np.array([20, 22, 17, 19, 18])
+
+# Fixed costs for each plant
+data_fixed_costs = np.array([12000, 15000, 17000, 13000, 16000])
+
+# Transportation costs per thousand units
+data_trans_costs = np.array(
+    [[4000, 2000, 3000, 2500, 4500],
+     [2500, 2600, 3400, 3000, 4000],
+     [1200, 1800, 2600, 4100, 3000],
+     [2200, 2600, 3100, 3700, 3200]]
+)
+
+# Indices over the plants and warehouses
+plant = Idx('plant', range=len(data_capacity))
+warehouse = Idx('warehouse', range=len(data_demand))
+
+# Indexed variables. Initial values become coefficients in the objective function.
+open = IndexedBase(Parameter('open', binary=True, value=data_fixed_costs))
+transport = IndexedBase(Parameter('transport', value=data_trans_costs))
+capacity = IndexedBase(Parameter('capacity'))
+demand = IndexedBase(Parameter('demand'))
```

Comment:
`IndexedBase(Parameter(...))`, or `IndexedParameter(...)`?

Reply:
Alright. I'm not sure familiar with sympy, especially this corner. If this is how sympy does its thing I agree that we should do the same.